### PR TITLE
TG-97: Implement edit classroom functionality

### PR DESCRIPTION
**Pull request by carlos-canedo-dev**
# Summary

Edit classroom and nested schedules

# Details

- Implement update classroom properties
- Implement update nested schedules
- Implement add new nested schedules
- Implement remove nested schedules
- Implement required requests to perform the classroom update

# Evidence

feature-tg-97-edit-classroom-fe-edit-classroom

### CODE REVIEW

**Review by RamiroJGuillen — APPROVED** [review]

File: src/environments/environment.development.ts [hunk 1]
```diff
@@ -3,8 +3,8 @@ export const environment = {
   apiBasePath: `https://localhost:5001/api`,
   resourceServerId: 'api',
   keycloak: {
-    issuer: 'https://localhost:8443/realms/Klockan',
-    redirectUri: 'https://localhost/',
+    issuer: 'http://localhost:8080/realms/Klockan',
```
Comment (Adrian-Mendoza-Jala) [inline src/environments/environment.development.ts]: Are you sure to change this config?

File: src/environments/environment.development.ts [hunk 2]
```diff
@@ -3,8 +3,8 @@ export const environment = {
...
+    redirectUri: 'https://localhost:4000/',
```
Comment (Adrian-Mendoza-Jala) [inline src/environments/environment.development.ts]: Same here

**Review by Adrian-Mendoza-Jala — CHANGES_REQUESTED** [review]

File: src/environments/environment.development.ts (hunk 1 above)
Comment (carlos-canedo-dev) [reply src/environments/environment.development.ts]: Fixed in [Restore development environment keycloak variables](https://github.com/JU-DEV-LatamBootcamp/Klockan-Frontend/pull/49/commits/99ed40139c61155193fd99bb019a9c5a0d23f08f) commit.

File: src/environments/environment.development.ts (hunk 2 above)
Comment (carlos-canedo-dev) [reply src/environments/environment.development.ts]: Fixed in [Restore development environment keycloak variables](https://github.com/JU-DEV-LatamBootcamp/Klockan-Frontend/pull/49/commits/99ed40139c61155193fd99bb019a9c5a0d23f08f) commit.

**Review by Adrian-Mendoza-Jala — APPROVED** [review]

**Pull request merged**